### PR TITLE
fix: style improvements

### PR DESCRIPTION
**Pull request by adrianHards**
run `rubocop` -a and `standardrb --fix` to better conventions as per our style guide

### CODE REVIEW

**Comment by CraigNorford** [comment]
Looks good, can you update confluence with the styling rules you've implemented here?

**Comment by adrianHards** [comment]
reviewing why tests failed

File: app/models/user.rb [hunk 1]
```diff
@@ -11,10 +11,10 @@ class User < ApplicationRecord
   enum :role, {mentor: 0, mentee: 1}
   devise :database_authenticatable, :registerable,
     :recoverable, :rememberable, :validatable
-# Creates profile and associates it to given user instance. Method only called upon initial user creation
+  # Creates profile and associates it to given user instance. Method only called upon initial user creation
   after_create :create_profile
 
-  private
```
Comment (adrianHards) [inline app/models/user.rb:17]: private does not make singleton methods private

**Review by SafiyahZ — APPROVED** [review]

**Comment by adrianHards** [comment]
"At least 1 approving review is required by reviewers with write access", does Safiyah not have write access?

**Review by CraigNorford — APPROVED** [review]

**Pull request merged**